packages/js/data/src/products/types.ts

```diff
@@ -28,7 +28,7 @@ export type ProductDownload = {
 	file: string;
 };
 
-export type ProductAttribute = {
+export type ProductProductAttribute = {
```

Comment:
Haha I guess this will do :) Not the most spectacular name, but does make sense.

Reply:
Agree :developers:! I'm afraid it would get funnier when renaming `ProductDownload`, etc :-D